src/test/java/org/osc/controller/nsc/OSGiIntegrationTest.java

```diff
@@ -67,28 +58,7 @@
 @ExamReactorStrategy(PerClass.class)
 public class OSGiIntegrationTest {
 
-    private static final String TEST_DB_URL_PREFIX = "jdbc:h2:";
-    private static final String TEST_DB_FILENAME = "./nscPlugin_OSGiIntegrationTest";
-    private static final String TEST_DB_URL_SUFFIX = ";MVCC\\=FALSE;LOCK_TIMEOUT\\=10000;MV_STORE=FALSE;";
-    private static final String TEST_DB_URL = TEST_DB_URL_PREFIX + TEST_DB_FILENAME + TEST_DB_URL_SUFFIX;
-
-    private static final String EADDR2_STR = "192.168.0.12";
-
-    private static final String EADDR1_STR = "192.168.0.11";
-
-    private static final String IADDR2_STR = "10.4.3.2";
-
-    private static final String IADDR1_STR = "10.4.3.1";
-
-    private static final String EMAC2_STR = "ee:ff:aa:bb:cc:02";
-
-    private static final String EMAC1_STR = "ee:ff:aa:bb:cc:01";
-
-    private static final String IMAC1_STR = "ff:ff:aa:bb:cc:01";
-
-    private static final String IMAC2_STR = "ff:ff:aa:bb:cc:02";
-
-    private static final String INSPMAC1_STR = "aa:aa:aa:bb:cc:01";
+    private static final String TEST_DB_URL = "jdbc:h2:mem:nsfc-mgr;DB_CLOSE_DELAY=-1";
```

Comment:
why do we want to keep the content of the db until the JVM shutsdown?

also DB name should be "nsc-osgi-db" not "nsfc"

also not used private field? Should result in eclipse warning...please make sure no warnings are introduced

Reply:
I want to be the one who clears the database. Without the DB_CLOSE_DELAY, the tests would not pass. em.close() does not work.

Renamed the databases.

redirApi? It was overshadowed. Fixed that.